roles/docker/tasks/cleanup.yml

```diff
@@ -12,3 +12,9 @@
   become: true
   with_items:
     - "{{ docker_api_port }}"
+
+- name: cleanup contiv-compose
```

Comment:
@mapuri - if we introduce contiv-compose --version, would it still not require a download to check the version? I am thinking then that we also need a release schedule to put the versioning on the image names?